pageserver/src/page_service.rs

```diff
@@ -422,19 +466,14 @@ impl PageServerHandler {
         pgb.write_message_noflush(&BeMessage::CopyInResponse)?;
         pgb.flush().await?;
 
-        let mut copyin_stream = Box::pin(copyin_stream(pgb));
+        let copyin_reader = StreamReader::new(copyin_stream(pgb));
+        tokio::pin!(copyin_reader);
```

Comment:
you can use now [std::pin ](https://blog.rust-lang.org/2023/03/09/Rust-1.68.0.html#local-pin-construction)

Reply:
Ooh, 1.68 feature, fresh off the oven :-). Fair point, but I think I'll push this as it is, and we can do a separate PR to change all our `tokio::pin!` calls.